setup.py

```diff
@@ -370,6 +370,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'pandas-gbq',
     pandas_requirement,
     'sqlalchemy-bigquery>=1.2.1',
+    'proto-plus>=1.19.6',
```

Comment:
```suggestion
    # A transient dependency of google-cloud-bigquery-datatransfer, but we
    # further constrain it since older versions are buggy.
    'proto-plus>=1.19.6',
```